server/src/main/java/com/epam/aidial/core/server/service/ApplicationService.java

```diff
@@ -307,7 +317,8 @@ public void copyApplication(ResourceDescriptor source, ResourceDescriptor destin
                     if (isPublicOrReview) {
                         throw new HttpException(HttpStatus.CONFLICT, "The application function must be deleted in public/review bucket");
                     }
-
+                    application.setCustomAppSchemaId(existing.getCustomAppSchemaId());
```

Comment:
Why do you copy it from existing into what you are going to copy and under this specific condition?